### PR TITLE
Fetch OIDC group and email claims dynamically

### PR DESCRIPTION
**Pull request by natanlao**
Depends on DataBiosphere/data-store#30

### CODE REVIEW

File: hca/util/__init__.py [hunk 1]
```diff
@@ -274,8 +274,18 @@ def __init__(self, config=None, swagger_url=None, **session_kwargs):
 
     @staticmethod
     def _audience(self):
-        return _deep_get(self.swagger_spec,["securityDefinitions","OauthSecurity","x-audience"]) or \
-               "https://auth.ucsc-cgp-redwood.org"
+        return _deep_get(self.swagger_spec, ["securityDefinitions", "OauthSecurity", "x-audience"]) or \
+            "https://auth.ucsc-cgp-redwood.org"
```
Comment (chmreid) [inline hca/util/__init__.py:278]: May want to grab `os.environ['OIDC_AUDIENCE']` instead of hard-coding this

**Review by chmreid — APPROVED** [review]
One nit, otherwise LGTM

File: hca/util/__init__.py (hunk 1 above)
Comment (amarjandu) [reply hca/util/__init__.py:278]: that `OIDC_AUDIENCE` env var is for the `data-store` repo, im not sure if we would want to have that external dependency here....... :gir

Comment (chmreid) [reply hca/util/__init__.py:278]: We can address this at a later time, merge away!

**Pull request merged**